backend/PythonClient/server/simulation_server.py

```diff
@@ -279,15 +295,41 @@ def stream(drone_name, camera_name):
             print(e)
             return "Error"
 
+@app.route('/uploadMission', methods=['POST'])
+def upload_file():
+    log_text = "" 
+    airsim_app = AirSimApplication()  # Create an instance of AirSimApplication
+
+    try:
+      file = request.files['file']
+      filename = file.filename
+      if not filename: 
+          raise ValueError("No file Provided");
+
+      custom_mission_dir = '../multirotor/mission/custom'
+      path = os.path.join(custom_mission_dir, filename)
+      file.save(path)
+      
+      #Code for Log success
+
+      airsim_app.append_pass_to_log(f"Report '{filename}' successfully uploaded at {path}.")
```

Comment:
The report doesn't seem to be showing up in the console. Might have a path issue. Everything else looks fine.